From a4ca7426f445e6c2a18282b451cced4300cc7f87 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Fri, 19 Jul 2024 11:30:01 +0200
Subject: [PATCH 1233/1695] dts: stm32: most stm32mp15 UARTs pinctrl are
 non-secure

On STM32MP15 based devices, UART2/3/4/5/6/7/8 cannot be secured.
Explicitly state that in the pinctrl nodes. This change ease the use
of a non-secure UART for OP-TEE output console on STM32MP15 based boards.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 480cd1827ad3fef8d3975126cd83c2a05d58967f dts: stm32: most stm32mp15 UARTs pinctrl are non-secure]
Change-Id: I57de8f64b21bdfedb1c5a24c7972fe2b266ab8c1
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/394350
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp15-pinctrl.dtsi | 184 +++++++++++------------
 1 file changed, 92 insertions(+), 92 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp15-pinctrl.dtsi b/core/arch/arm/dts/stm32mp15-pinctrl.dtsi
index 7edff9ca8..acd12c5ae 100644
--- a/core/arch/arm/dts/stm32mp15-pinctrl.dtsi
+++ b/core/arch/arm/dts/stm32mp15-pinctrl.dtsi
@@ -1830,356 +1830,356 @@
 
 	uart4_pins_a: uart4-0 {
 		pins1 {
-			pinmux = <STM32_PINMUX('G', 11, AF6)>; /* UART4_TX */
+			pinmux = <STM32_PINMUX_NSEC('G', 11, AF6)>; /* UART4_TX */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('B', 2, AF8)>; /* UART4_RX */
+			pinmux = <STM32_PINMUX_NSEC('B', 2, AF8)>; /* UART4_RX */
 			bias-disable;
 		};
 	};
 
 	uart4_idle_pins_a: uart4-idle-0 {
 		pins1 {
-			pinmux = <STM32_PINMUX('G', 11, ANALOG)>; /* UART4_TX */
+			pinmux = <STM32_PINMUX_NSEC('G', 11, ANALOG)>; /* UART4_TX */
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('B', 2, AF8)>; /* UART4_RX */
+			pinmux = <STM32_PINMUX_NSEC('B', 2, AF8)>; /* UART4_RX */
 			bias-disable;
 		};
 	};
 
 	uart4_sleep_pins_a: uart4-sleep-0 {
 		pins {
-			pinmux = <STM32_PINMUX('G', 11, ANALOG)>, /* UART4_TX */
-				 <STM32_PINMUX('B', 2, ANALOG)>; /* UART4_RX */
+			pinmux = <STM32_PINMUX_NSEC('G', 11, ANALOG)>, /* UART4_TX */
+				 <STM32_PINMUX_NSEC('B', 2, ANALOG)>; /* UART4_RX */
 		};
 	};
 
 	uart4_pins_b: uart4-1 {
 		pins1 {
-			pinmux = <STM32_PINMUX('D', 1, AF8)>; /* UART4_TX */
+			pinmux = <STM32_PINMUX_NSEC('D', 1, AF8)>; /* UART4_TX */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('B', 2, AF8)>; /* UART4_RX */
+			pinmux = <STM32_PINMUX_NSEC('B', 2, AF8)>; /* UART4_RX */
 			bias-disable;
 		};
 	};
 
 	uart4_pins_c: uart4-2 {
 		pins1 {
-			pinmux = <STM32_PINMUX('G', 11, AF6)>; /* UART4_TX */
+			pinmux = <STM32_PINMUX_NSEC('G', 11, AF6)>; /* UART4_TX */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('B', 2, AF8)>; /* UART4_RX */
+			pinmux = <STM32_PINMUX_NSEC('B', 2, AF8)>; /* UART4_RX */
 			bias-disable;
 		};
 	};
 
 	uart7_pins_a: uart7-0 {
 		pins1 {
-			pinmux = <STM32_PINMUX('E', 8, AF7)>; /* UART7_TX */
+			pinmux = <STM32_PINMUX_NSEC('E', 8, AF7)>; /* UART7_TX */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('E', 7, AF7)>, /* UART7_RX */
-				 <STM32_PINMUX('E', 10, AF7)>, /* UART7_CTS */
-				 <STM32_PINMUX('E', 9, AF7)>; /* UART7_RTS */
+			pinmux = <STM32_PINMUX_NSEC('E', 7, AF7)>, /* UART7_RX */
+				 <STM32_PINMUX_NSEC('E', 10, AF7)>, /* UART7_CTS */
+				 <STM32_PINMUX_NSEC('E', 9, AF7)>; /* UART7_RTS */
 			bias-disable;
 		};
 	};
 
 	uart7_pins_b: uart7-1 {
 		pins1 {
-			pinmux = <STM32_PINMUX('F', 7, AF7)>; /* UART7_TX */
+			pinmux = <STM32_PINMUX_NSEC('F', 7, AF7)>; /* UART7_TX */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('F', 6, AF7)>; /* UART7_RX */
+			pinmux = <STM32_PINMUX_NSEC('F', 6, AF7)>; /* UART7_RX */
 			bias-disable;
 		};
 	};
 
 	uart7_pins_c: uart7-2 {
 		pins1 {
-			pinmux = <STM32_PINMUX('E', 8, AF7)>; /* UART7_TX */
+			pinmux = <STM32_PINMUX_NSEC('E', 8, AF7)>; /* UART7_TX */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('E', 7, AF7)>; /* UART7_RX */
+			pinmux = <STM32_PINMUX_NSEC('E', 7, AF7)>; /* UART7_RX */
 			bias-pull-up;
 		};
 	};
 
 	uart7_idle_pins_c: uart7-idle-2 {
 		pins1 {
-			pinmux = <STM32_PINMUX('E', 8, ANALOG)>; /* UART7_TX */
+			pinmux = <STM32_PINMUX_NSEC('E', 8, ANALOG)>; /* UART7_TX */
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('E', 7, AF7)>; /* UART7_RX */
+			pinmux = <STM32_PINMUX_NSEC('E', 7, AF7)>; /* UART7_RX */
 			bias-pull-up;
 		};
 	};
 
 	uart7_sleep_pins_c: uart7-sleep-2 {
 		pins {
-			pinmux = <STM32_PINMUX('E', 8, ANALOG)>, /* UART7_TX */
-				 <STM32_PINMUX('E', 7, ANALOG)>; /* UART7_RX */
+			pinmux = <STM32_PINMUX_NSEC('E', 8, ANALOG)>, /* UART7_TX */
+				 <STM32_PINMUX_NSEC('E', 7, ANALOG)>; /* UART7_RX */
 		};
 	};
 
 	uart8_pins_a: uart8-0 {
 		pins1 {
-			pinmux = <STM32_PINMUX('E', 1, AF8)>; /* UART8_TX */
+			pinmux = <STM32_PINMUX_NSEC('E', 1, AF8)>; /* UART8_TX */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('E', 0, AF8)>; /* UART8_RX */
+			pinmux = <STM32_PINMUX_NSEC('E', 0, AF8)>; /* UART8_RX */
 			bias-disable;
 		};
 	};
 
 	uart8_rtscts_pins_a: uart8rtscts-0 {
 		pins {
-			pinmux = <STM32_PINMUX('G', 7, AF8)>, /* UART8_RTS */
-				 <STM32_PINMUX('G', 10, AF8)>; /* UART8_CTS */
+			pinmux = <STM32_PINMUX_NSEC('G', 7, AF8)>, /* UART8_RTS */
+				 <STM32_PINMUX_NSEC('G', 10, AF8)>; /* UART8_CTS */
 			bias-disable;
 		};
 	};
 
 	usart2_pins_a: usart2-0 {
 		pins1 {
-			pinmux = <STM32_PINMUX('F', 5, AF7)>, /* USART2_TX */
-				 <STM32_PINMUX('D', 4, AF7)>; /* USART2_RTS */
+			pinmux = <STM32_PINMUX_NSEC('F', 5, AF7)>, /* USART2_TX */
+				 <STM32_PINMUX_NSEC('D', 4, AF7)>; /* USART2_RTS */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('D', 6, AF7)>, /* USART2_RX */
-				 <STM32_PINMUX('D', 3, AF7)>; /* USART2_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('D', 6, AF7)>, /* USART2_RX */
+				 <STM32_PINMUX_NSEC('D', 3, AF7)>; /* USART2_CTS_NSS */
 			bias-disable;
 		};
 	};
 
 	usart2_sleep_pins_a: usart2-sleep-0 {
 		pins {
-			pinmux = <STM32_PINMUX('F', 5, ANALOG)>, /* USART2_TX */
-				 <STM32_PINMUX('D', 4, ANALOG)>, /* USART2_RTS */
-				 <STM32_PINMUX('D', 6, ANALOG)>, /* USART2_RX */
-				 <STM32_PINMUX('D', 3, ANALOG)>; /* USART2_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('F', 5, ANALOG)>, /* USART2_TX */
+				 <STM32_PINMUX_NSEC('D', 4, ANALOG)>, /* USART2_RTS */
+				 <STM32_PINMUX_NSEC('D', 6, ANALOG)>, /* USART2_RX */
+				 <STM32_PINMUX_NSEC('D', 3, ANALOG)>; /* USART2_CTS_NSS */
 		};
 	};
 
 	usart2_pins_b: usart2-1 {
 		pins1 {
-			pinmux = <STM32_PINMUX('F', 5, AF7)>, /* USART2_TX */
-				 <STM32_PINMUX('A', 1, AF7)>; /* USART2_RTS */
+			pinmux = <STM32_PINMUX_NSEC('F', 5, AF7)>, /* USART2_TX */
+				 <STM32_PINMUX_NSEC('A', 1, AF7)>; /* USART2_RTS */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('F', 4, AF7)>, /* USART2_RX */
-				 <STM32_PINMUX('E', 15, AF7)>; /* USART2_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('F', 4, AF7)>, /* USART2_RX */
+				 <STM32_PINMUX_NSEC('E', 15, AF7)>; /* USART2_CTS_NSS */
 			bias-disable;
 		};
 	};
 
 	usart2_sleep_pins_b: usart2-sleep-1 {
 		pins {
-			pinmux = <STM32_PINMUX('F', 5, ANALOG)>, /* USART2_TX */
-				 <STM32_PINMUX('A', 1, ANALOG)>, /* USART2_RTS */
-				 <STM32_PINMUX('F', 4, ANALOG)>, /* USART2_RX */
-				 <STM32_PINMUX('E', 15, ANALOG)>; /* USART2_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('F', 5, ANALOG)>, /* USART2_TX */
+				 <STM32_PINMUX_NSEC('A', 1, ANALOG)>, /* USART2_RTS */
+				 <STM32_PINMUX_NSEC('F', 4, ANALOG)>, /* USART2_RX */
+				 <STM32_PINMUX_NSEC('E', 15, ANALOG)>; /* USART2_CTS_NSS */
 		};
 	};
 
 	usart2_pins_c: usart2-2 {
 		pins1 {
-			pinmux = <STM32_PINMUX('D', 5, AF7)>, /* USART2_TX */
-				 <STM32_PINMUX('D', 4, AF7)>; /* USART2_RTS */
+			pinmux = <STM32_PINMUX_NSEC('D', 5, AF7)>, /* USART2_TX */
+				 <STM32_PINMUX_NSEC('D', 4, AF7)>; /* USART2_RTS */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('D', 6, AF7)>, /* USART2_RX */
-				 <STM32_PINMUX('D', 3, AF7)>; /* USART2_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('D', 6, AF7)>, /* USART2_RX */
+				 <STM32_PINMUX_NSEC('D', 3, AF7)>; /* USART2_CTS_NSS */
 			bias-disable;
 		};
 	};
 
 	usart2_idle_pins_c: usart2-idle-2 {
 		pins1 {
-			pinmux = <STM32_PINMUX('D', 5, ANALOG)>, /* USART2_TX */
-				 <STM32_PINMUX('D', 3, ANALOG)>; /* USART2_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('D', 5, ANALOG)>, /* USART2_TX */
+				 <STM32_PINMUX_NSEC('D', 3, ANALOG)>; /* USART2_CTS_NSS */
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('D', 4, AF7)>; /* USART2_RTS */
+			pinmux = <STM32_PINMUX_NSEC('D', 4, AF7)>; /* USART2_RTS */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins3 {
-			pinmux = <STM32_PINMUX('D', 6, AF7)>; /* USART2_RX */
+			pinmux = <STM32_PINMUX_NSEC('D', 6, AF7)>; /* USART2_RX */
 			bias-disable;
 		};
 	};
 
 	usart2_sleep_pins_c: usart2-sleep-2 {
 		pins {
-			pinmux = <STM32_PINMUX('D', 5, ANALOG)>, /* USART2_TX */
-				 <STM32_PINMUX('D', 4, ANALOG)>, /* USART2_RTS */
-				 <STM32_PINMUX('D', 6, ANALOG)>, /* USART2_RX */
-				 <STM32_PINMUX('D', 3, ANALOG)>; /* USART2_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('D', 5, ANALOG)>, /* USART2_TX */
+				 <STM32_PINMUX_NSEC('D', 4, ANALOG)>, /* USART2_RTS */
+				 <STM32_PINMUX_NSEC('D', 6, ANALOG)>, /* USART2_RX */
+				 <STM32_PINMUX_NSEC('D', 3, ANALOG)>; /* USART2_CTS_NSS */
 		};
 	};
 
 	usart3_pins_a: usart3-0 {
 		pins1 {
-			pinmux = <STM32_PINMUX('B', 10, AF7)>; /* USART3_TX */
+			pinmux = <STM32_PINMUX_NSEC('B', 10, AF7)>; /* USART3_TX */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('B', 12, AF8)>; /* USART3_RX */
+			pinmux = <STM32_PINMUX_NSEC('B', 12, AF8)>; /* USART3_RX */
 			bias-disable;
 		};
 	};
 
 	usart3_pins_b: usart3-1 {
 		pins1 {
-			pinmux = <STM32_PINMUX('B', 10, AF7)>, /* USART3_TX */
-				 <STM32_PINMUX('G', 8, AF8)>; /* USART3_RTS */
+			pinmux = <STM32_PINMUX_NSEC('B', 10, AF7)>, /* USART3_TX */
+				 <STM32_PINMUX_NSEC('G', 8, AF8)>; /* USART3_RTS */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('B', 12, AF8)>, /* USART3_RX */
-				 <STM32_PINMUX('I', 10, AF8)>; /* USART3_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('B', 12, AF8)>, /* USART3_RX */
+				 <STM32_PINMUX_NSEC('I', 10, AF8)>; /* USART3_CTS_NSS */
 			bias-pull-up;
 		};
 	};
 
 	usart3_idle_pins_b: usart3-idle-1 {
 		pins1 {
-			pinmux = <STM32_PINMUX('B', 10, ANALOG)>, /* USART3_TX */
-				 <STM32_PINMUX('I', 10, ANALOG)>; /* USART3_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('B', 10, ANALOG)>, /* USART3_TX */
+				 <STM32_PINMUX_NSEC('I', 10, ANALOG)>; /* USART3_CTS_NSS */
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('G', 8, AF8)>; /* USART3_RTS */
+			pinmux = <STM32_PINMUX_NSEC('G', 8, AF8)>; /* USART3_RTS */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins3 {
-			pinmux = <STM32_PINMUX('B', 12, AF8)>; /* USART3_RX */
+			pinmux = <STM32_PINMUX_NSEC('B', 12, AF8)>; /* USART3_RX */
 			bias-pull-up;
 		};
 	};
 
 	usart3_sleep_pins_b: usart3-sleep-1 {
 		pins {
-			pinmux = <STM32_PINMUX('B', 10, ANALOG)>, /* USART3_TX */
-				 <STM32_PINMUX('G', 8, ANALOG)>, /* USART3_RTS */
-				 <STM32_PINMUX('I', 10, ANALOG)>, /* USART3_CTS_NSS */
-				 <STM32_PINMUX('B', 12, ANALOG)>; /* USART3_RX */
+			pinmux = <STM32_PINMUX_NSEC('B', 10, ANALOG)>, /* USART3_TX */
+				 <STM32_PINMUX_NSEC('G', 8, ANALOG)>, /* USART3_RTS */
+				 <STM32_PINMUX_NSEC('I', 10, ANALOG)>, /* USART3_CTS_NSS */
+				 <STM32_PINMUX_NSEC('B', 12, ANALOG)>; /* USART3_RX */
 		};
 	};
 
 	usart3_pins_c: usart3-2 {
 		pins1 {
-			pinmux = <STM32_PINMUX('B', 10, AF7)>, /* USART3_TX */
-				 <STM32_PINMUX('G', 8, AF8)>; /* USART3_RTS */
+			pinmux = <STM32_PINMUX_NSEC('B', 10, AF7)>, /* USART3_TX */
+				 <STM32_PINMUX_NSEC('G', 8, AF8)>; /* USART3_RTS */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('B', 12, AF8)>, /* USART3_RX */
-				 <STM32_PINMUX('B', 13, AF7)>; /* USART3_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('B', 12, AF8)>, /* USART3_RX */
+				 <STM32_PINMUX_NSEC('B', 13, AF7)>; /* USART3_CTS_NSS */
 			bias-pull-up;
 		};
 	};
 
 	usart3_idle_pins_c: usart3-idle-2 {
 		pins1 {
-			pinmux = <STM32_PINMUX('B', 10, ANALOG)>, /* USART3_TX */
-				 <STM32_PINMUX('B', 13, ANALOG)>; /* USART3_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('B', 10, ANALOG)>, /* USART3_TX */
+				 <STM32_PINMUX_NSEC('B', 13, ANALOG)>; /* USART3_CTS_NSS */
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('G', 8, AF8)>; /* USART3_RTS */
+			pinmux = <STM32_PINMUX_NSEC('G', 8, AF8)>; /* USART3_RTS */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins3 {
-			pinmux = <STM32_PINMUX('B', 12, AF8)>; /* USART3_RX */
+			pinmux = <STM32_PINMUX_NSEC('B', 12, AF8)>; /* USART3_RX */
 			bias-pull-up;
 		};
 	};
 
 	usart3_sleep_pins_c: usart3-sleep-2 {
 		pins {
-			pinmux = <STM32_PINMUX('B', 10, ANALOG)>, /* USART3_TX */
-				 <STM32_PINMUX('G', 8, ANALOG)>, /* USART3_RTS */
-				 <STM32_PINMUX('B', 13, ANALOG)>, /* USART3_CTS_NSS */
-				 <STM32_PINMUX('B', 12, ANALOG)>; /* USART3_RX */
+			pinmux = <STM32_PINMUX_NSEC('B', 10, ANALOG)>, /* USART3_TX */
+				 <STM32_PINMUX_NSEC('G', 8, ANALOG)>, /* USART3_RTS */
+				 <STM32_PINMUX_NSEC('B', 13, ANALOG)>, /* USART3_CTS_NSS */
+				 <STM32_PINMUX_NSEC('B', 12, ANALOG)>; /* USART3_RX */
 		};
 	};
 
 	usart3_pins_d: usart3-3 {
 		pins1 {
-			pinmux = <STM32_PINMUX('B', 10, AF7)>, /* USART3_TX */
-				 <STM32_PINMUX('G', 8, AF8)>; /* USART3_RTS */
+			pinmux = <STM32_PINMUX_NSEC('B', 10, AF7)>, /* USART3_TX */
+				 <STM32_PINMUX_NSEC('G', 8, AF8)>; /* USART3_RTS */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('D', 9, AF7)>, /* USART3_RX */
-				 <STM32_PINMUX('D', 11, AF7)>; /* USART3_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('D', 9, AF7)>, /* USART3_RX */
+				 <STM32_PINMUX_NSEC('D', 11, AF7)>; /* USART3_CTS_NSS */
 			bias-disable;
 		};
 	};
 
 	usart3_idle_pins_d: usart3-idle-3 {
 		pins1 {
-			pinmux = <STM32_PINMUX('B', 10, ANALOG)>, /* USART3_TX */
-				 <STM32_PINMUX('G', 8, ANALOG)>, /* USART3_RTS */
-				 <STM32_PINMUX('D', 11, ANALOG)>; /* USART3_CTS_NSS */
+			pinmux = <STM32_PINMUX_NSEC('B', 10, ANALOG)>, /* USART3_TX */
+				 <STM32_PINMUX_NSEC('G', 8, ANALOG)>, /* USART3_RTS */
+				 <STM32_PINMUX_NSEC('D', 11, ANALOG)>; /* USART3_CTS_NSS */
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('D', 9, AF7)>; /* USART3_RX */
+			pinmux = <STM32_PINMUX_NSEC('D', 9, AF7)>; /* USART3_RX */
 			bias-disable;
 		};
 	};
 
 	usart3_sleep_pins_d: usart3-sleep-3 {
 		pins {
-			pinmux = <STM32_PINMUX('B', 10, ANALOG)>, /* USART3_TX */
-				 <STM32_PINMUX('G', 8, ANALOG)>, /* USART3_RTS */
-				 <STM32_PINMUX('D', 11, ANALOG)>, /* USART3_CTS_NSS */
-				 <STM32_PINMUX('D', 9, ANALOG)>; /* USART3_RX */
+			pinmux = <STM32_PINMUX_NSEC('B', 10, ANALOG)>, /* USART3_TX */
+				 <STM32_PINMUX_NSEC('G', 8, ANALOG)>, /* USART3_RTS */
+				 <STM32_PINMUX_NSEC('D', 11, ANALOG)>, /* USART3_CTS_NSS */
+				 <STM32_PINMUX_NSEC('D', 9, ANALOG)>; /* USART3_RX */
 		};
 	};
 
-- 
2.47.2

